From 5d4f0cafc45ea7f10d15c665ed0027c463baaf3c Mon Sep 17 00:00:00 2001
From: Esther Z <Esther.Z@linux.alibaba.com>
Date: Sun, 16 Jun 2024 14:42:54 +0800
Subject: [PATCH 098/129] dt-bindings: add AON resource id headfile

The AON (Always-On) firmware manages system resource service.
Within an AON-integrated system, each component or device is assigned
a unique resource identifier.
These identifiers serve as a means for the AON firmware to facilitate
and control services, such as power management of different power domains.

Signed-off-by: Esther Z<Esther.Z@linux.alibaba.com>
---
 include/dt-bindings/firmware/thead/rsrc.h | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)
 create mode 100644 include/dt-bindings/firmware/thead/rsrc.h

--- /dev/null
+++ b/include/dt-bindings/firmware/thead/rsrc.h
@@ -0,0 +1,18 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+/*
+ * Copyright (C) 2022 Alibaba Group Holding Limited.
+ */
+
+#ifndef __DT_BINDINGS_RSCRC_TH1520_H
+#define __DT_BINDINGS_RSCRC_TH1520_H
+
+#define TH1520_AON_AUDIO_PD	0
+#define TH1520_AON_VDEC_PD	1
+#define TH1520_AON_NPU_PD	2
+#define TH1520_AON_VENC_PD	3
+#define TH1520_AON_GPU_PD	4
+#define TH1520_AON_DSP0_PD	5
+#define TH1520_AON_DSP1_PD	6
+#define TH1520_AON_R_LAST	7
+
+#endif
